testing/go/dolt_functions_test.go

```diff
@@ -283,5 +282,126 @@ func TestDoltFunctions(t *testing.T) {
 				},
 			},
 		},
+		{
+			Name: "smoke test select dolt_diff functions and tables for multiple schemas",
+			SetUpScript: []string{
+				"CREATE TABLE t1 (pk int primary key);",
+				"INSERT INTO t1 VALUES (1);",
+				"CREATE SCHEMA testschema;",
+				"CREATE TABLE testschema.t2 (pk int primary key);",
+				"INSERT INTO testschema.t2 VALUES (1);",
+			},
+			Assertions: []ScriptTestAssertion{
+				{
+					Query: "SELECT * FROM dolt_status;",
+					Expected: []sql.Row{
+						{"public.t1", 0, "new table"},
+						{"testschema.t2", 0, "new table"},
+						{"testschema", 0, "new schema"},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff_stat('HEAD', 'WORKING')",
+					Expected: []sql.Row{
+						{"public.t1", 0, 1, 0, 0, 1, 0, 0, 0, 1, 0, 1},
+						{"testschema.t2", 0, 1, 0, 0, 1, 0, 0, 0, 1, 0, 1},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff_stat('HEAD', 'WORKING', 't1')",
+					Expected: []sql.Row{
+						{"public.t1", 0, 1, 0, 0, 1, 0, 0, 0, 1, 0, 1},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff_stat('HEAD', 'WORKING', 't2')",
+					Expected: []sql.Row{
+						{"testschema.t2", 0, 1, 0, 0, 1, 0, 0, 0, 1, 0, 1},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff_summary('HEAD', 'WORKING')",
+					Expected: []sql.Row{
+						{"", "public.t1", "added", 1, 1},
+						{"", "testschema.t2", "added", 1, 1},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff_summary('HEAD', 'WORKING', 't1')",
+					Expected: []sql.Row{
+						{"", "public.t1", "added", 1, 1},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff_summary('HEAD', 'WORKING', 't2')",
+					Expected: []sql.Row{
+						{"", "testschema.t2", "added", 1, 1},
+					},
+				},
+				{
+					Query: "SELECT diff_type, from_pk, to_pk FROM dolt_diff('HEAD', 'WORKING', 't1')",
+					Expected: []sql.Row{
+						{"added", nil, 1},
+					},
+				},
+				{
+					Query: "SELECT diff_type, from_pk, to_pk FROM dolt_diff('HEAD', 'WORKING', 't2')",
+					Expected: []sql.Row{
+						{"added", nil, 1},
+					},
+				},
+				{
+					Skip:  true, // TODO: dolt_commit_diff_* tables must be filtered to a single 'to_commit'
+					Query: "SELECT diff_type, from_pk, to_pk FROM dolt_commit_diff_t1 WHERE to_commit=HASHOF('main') AND from_commit='WORKING'",
+					Expected: []sql.Row{
+						{"added", nil, 1},
+					},
+				},
+				{
+					Query: "SELECT * FROM dolt_diff",
+					Expected: []sql.Row{
+						{"WORKING", "public.t1", nil, nil, nil, nil, 1, 1},
+						{"WORKING", "testschema.t2", nil, nil, nil, nil, 1, 1},
+					},
+				},
+				{
+					Skip:  true,
```

Comment:
Adding a comment for the current failure for each skip is helpful

Reply:
Oops meant to unskip these